src/components/QueryBuilder.vue

```diff
@@ -58,9 +63,18 @@ export default Vue.extend( {
 				property: null as null | Error,
 				value: null as null | Error,
 			},
+			optionItems: [
+				{ value: 'without' },
+				{ value: 'with' },
+				{ value: 'regardless' },
```

Comment:
It looks good. We probably need to check if it is with these options or "matching" and "regardless of value" as indicated in this ticket. https://phabricator.wikimedia.org/T265245

Reply:
got an updated from charlie and adjusted it accordingly